eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/controller/TopicController.java

```diff
@@ -59,22 +59,22 @@ public ResponseEntity<Object> preflight() {
     }
 
     @CrossOrigin
-    @GetMapping
+    @GetMapping("/getList")
     public Result<List<TopicProperties>> getList() {
-        List<TopicProperties> topicList = topicCore.getTopic();
+        List<TopicProperties> topicList = topicCore.getTopics();
         return Result.success(topicList);
     }
 
     @CrossOrigin
-    @PostMapping
+    @PostMapping("/create")
     public Result<Object> create(@RequestBody CreateTopicRequest createTopicRequest) {
         String topicName = createTopicRequest.getName();
         topicCore.createTopic(topicName);
         return Result.success();
     }
 
     @CrossOrigin
-    @DeleteMapping
+    @DeleteMapping("delete")
     public Result<Object> delete(@RequestBody DeleteTopicRequest deleteTopicRequest) {
```

Comment:
The `list`, `new`, `delete` endpoint name is more standard. 

Fill in your missing `/` symbols.

Reply:
Since you have used `@GetMapping`, `@PostMapping` and `@DeleteMapping` for these three endpoints, then an additional endpoint name is unnecessary.

By the way, `@PutMapping` is suitable for update operations.

Reply:
明白了